server/utils/telemetry/rudder.go

```diff
@@ -0,0 +1,38 @@
+package telemetry
+
+import rudder "github.com/rudderlabs/analytics-go"
+
+const (
+	rudderDataPlaneURL = ""
+	rudderWriteKey     = ""
+)
```

Comment:
What is the short and the mid term solution to get this information? Can we squeeze a new RPC API method into 5.24? Or hard code it given that it won't change?

Reply:
The plan here is to have it hard coded. I still need to discuss with the analytics team to decide exactly what values we will use here, but that is the plan.

It is not my favorite choice, but make things incredibly simpler, and it seems it is an acceptable risk (NPS plugin is doing a similar thing).